packages/attribute-slicer-powerbi/src/state.ts

```diff
@@ -120,50 +140,32 @@ export default class AttributeSlicerVisualState extends HasSettings implements I
      * Hide Blank items
      */
     @bool({
+        persist: false,
         category: "Display",
         displayName: "Hide Empty Items",
         description: "Hide empty / blank Items.",
         defaultValue: false,
     })
     public hideEmptyItems?: boolean;
 
-    /**
-     * The list of selected items
-     */
-    @setting({
-        name: "selection",
-        displayName: "Selection",
-        hidden: true,
-        config: {
-            type: { text: {} },
-        },
-        parse: (v, d, dv) => parseSelectionFromPBI(dv),
-        compose: (value, d) => convertSelectionToPBI(value),
-    })
-    public selectedItems?: {
-        id: any;
-        match: any;
-        value: any;
-        renderedValue?: any;
-        selector: any;
-    }[];
-
     /**
      * The text size in pt
      */
     @setting({
+        persist: false,
         displayName: "Text Size",
         description: "The size of the text",
         defaultValue: DEFAULT_STATE.textSize,
-        parse: val => val ? PixelConverter.fromPointToPixel(parseFloat(val)) : DEFAULT_STATE.textSize,
-        compose: val => PixelConverter.toPoint(val ? val : DEFAULT_STATE.textSize),
+        parse: val => val ? type.PixelConverter.fromPointToPixel(parseFloat(val)) : DEFAULT_STATE.textSize,
+        compose: val => type.PixelConverter.toPoint(val ? val : DEFAULT_STATE.textSize),
     })
     public textSize?: number;
 
    /**
     * The font color used to display item text
     */
    @color({
+        persist: false,
```

Comment:
Why is persist false for all these settings?  Does that mean if you save and close the workbook and open it again you'll loose these settings?

Reply:
The user can still tweak these via the PBI formatting pane, and they will still save, we just don't persist changes back to powerbi. So, if for some reason, the visual does `settings.textSize = 16`, it won't save that back to powerbi.  In effect it just makes it one way, from `PBI -> Visual`, but not `Visual -> PBI`, if that makes sense.

Reply:
ah, yes that does make sense.  Thanks